src/app/shared/plunker/plunker-button.ts

```diff
@@ -6,18 +6,30 @@ import { ExampleData } from '../../examples/example-data';
   selector: 'plunker-button',
   templateUrl: './plunker-button.html',
   providers: [PlunkerWriter],
+  host: {
+    '(mouseover)': 'isDisabled = !plunkerForm'
+  }
 })
 export class PlunkerButton {
-  exampleData: ExampleData;
+  /**
+   * The button becomes disabled if the user hovers over the button before the plunker form
+   * is created. After the form is created, the button becomes enabled again.
+   */
+  isDisabled: boolean = false;
```

Comment:
Initially I had it disabled by default and only enabled when the form was created. However, this caused a bit of a flicker on the button where it goes from disabled -> enabled right after the page loads.

With the current implementation, it always appears active to the user except in the small chance they manage to mouse over it to interact with it. In which case, it disables itself and shows a tooltip saying that we building the form.